presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParser.java

```diff
@@ -1626,6 +1630,20 @@ public void testDropColumn()
         assertStatement("ALTER TABLE IF EXISTS foo.t DROP COLUMN IF EXISTS c", new DropColumn(QualifiedName.of("foo", "t"), identifier("c"), true, true));
     }
 
+    @Test
+    public void testAlterTableSetAuthorization()
+    {
+        assertStatement(
+                "ALTER TABLE foo.bar.baz SET AUTHORIZATION qux",
```

Comment:
s/qux/testing_user
s/foo.bar.baz/tesing_catalog.testing_schema.testing_table

Reply:
The whole class is filled with foos and bars. Using something else here will be inconsistent.

Reply:
maybe it's better to not add to this foo.bar.baz habbit then?

Reply:
I highly disagree. Holding a bad (in your opinion) convention is better than breaking it.